src/amber/environment/settings.cr

```diff
@@ -1,10 +1,18 @@
-require "logger"
+require "./logger"
 require "yaml"
 
 module Amber::Environment
   class Settings
-    property logger : ::Logger = Logger.new(STDOUT)
-    property colorize_logging : Bool
+    alias LoggingType = NamedTuple(
+      severity: String,
+      color: Bool,
```

Comment:
Color should probably be "show_color" since `color` seems like the value should be `blue` or something instead of true/false.

Reply:
Maybe not `color` nor `show_color` but `colorize` ? 😅